frontend/src/lib/components/displays/LearningUnitDisplay.svelte

```diff
@@ -0,0 +1,42 @@
+<script lang="ts">
+    import { AlignLeft, GripVertical } from 'lucide-svelte';
+    import {browserApiClient} from "$lib/api/browserApiClient.js";
+    import {getLearningUnitById, deleteLearningUnit, regenerateLearningUnit} from "$lib/api/collections/learningUnit";
+    import {goto} from "$app/navigation";
+
+    const {learningUnit} = $props();
+
+    async function openLearningUnit() {
```

Comment:
You did NOT think ONE second about what happens here and very clearly just accepted AIs garbage... please understand your code and if you use AI ALWAYS read it first.